src/api/app/jobs/send_event_emails_job.rb

```diff
@@ -2,6 +2,7 @@ class SendEventEmailsJob < ApplicationJob
   queue_as :mailers
 
   def perform
+    User.current = User.find_nobody!
```

Comment:
[nitpick] I think it would be better to consider the case the user is `nil` or ensure the user is always set instead of assigning it all over the code.... but it may be too late for this already... :sob:

Reply:
> [nitpick] I think it would be better to consider the case the user is `nil` or ensure the user is always set instead of assigning it all over the code....

Do you mean to ensure the user is always set for all the jobs?

Can we ensure the user here in this job, and when needed, add it in all other jobs in other pull requests,... and then refactor the code if also needed?

Reply:
> always set instead of assigning it all over the code.... but it may be too late for this already...

You are a couple of years to late. Authentication, session handling and authorization needs some serious refactoring. But not here.

Reply:
@hennevogel 

> You are a couple of years to late. Authentication, session handling and authorization needs some serious refactoring. But not here.

That is exactly what I meant....